tests/e2e/support/api/keycloak/user.ts

```diff
@@ -67,7 +68,8 @@ export const assignRole = async ({
   uuid: string
   role: string
 }) => {
-  return request({
+  // can assign multiple realm role at once
+  return await request({
```

Comment:
I don't see reason for using `await` but not a big deal
```suggestion
  return request({
```